ocrd_utils/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Pillow >= 5.3.0
```

Comment:
Maybe this needs to be `>= 6.1.0` – see #270.

Reply:
I'll debug that separately.